mx.truffleruby/mx_truffleruby.py

```diff
@@ -124,6 +125,10 @@ def needsBuild(self, newestInput):
     def build(self):
         self.store_jvm_args()
 
+        try:
+            os.unlink(self.launcher)
```

Comment:
@gilles-duboscq Shouldn't the `clean` method already do that?
Maybe the problem happens if the symlink points to a non-existent file?
Then I think we should just move the `unlink` to `clean()`.

Reply:
Looks like `os.path.lexists()` could be used instead of try/catch.